client/src/components/Allocations/AllocationsModals/AllocationsTeamViewModal/AllocationsTeamViewModal.jsx

```diff
@@ -43,6 +36,43 @@ const AllocationsTeamViewModal = ({
   const resetCard = () => {
     setCard(null);
   };
+
+  useEffect(() => {
+    dispatch({
+      type: 'ALLOCATION_OPERATION_REMOVE_USER_INIT',
+    });
+  }, [isOpen]);
+
+  const onAdd = useCallback(
+    (newUser) => {
+      dispatch({
+        type: 'ADD_USER_TO_TAS_PROJECT',
+        payload: {
+          projectId,
+          id: newUser.user.username,
+          projectName,
```

Comment:
Got an error trying to add a user:
![Screen Shot 2022-05-25 at 5 10 13 PM](https://user-images.githubusercontent.com/20326896/170376376-08392140-a5ea-4226-8f8c-c8d0cf6a169f.png)


```
Uncaught ReferenceError: projectName is not defined
```